tests/e2e-tests/config/default.json

```diff
@@ -9,5 +9,46 @@
       "username": "customer",
       "password": "password"
     }
+  },
+  "addresses": {
+    "admin": {
+      "store": {
+        "firstname": "John",
+        "lastname": "Doe",
+        "company": "Automattic",
+        "country": "United States (US)",
+        "addressfirstline": "addr 1",
+        "addresssecondline": "addr 2",
+        "city": "San Francisco",
+        "state": "CA",
+        "postcode": "94107"
+      }
+    },
+    "customer": {
+      "billing": {
+        "firstname": "John",
+        "lastname": "Doe",
+        "company": "Automattic",
+        "country": "United States (US)",
+        "addressfirstline": "addr 1",
+        "addresssecondline": "addr 2",
+        "city": "San Francisco",
+        "state": "CA",
+        "postcode": "94107",
+        "phone": "123456789",
+        "email": "john.doe@example.com"
+      },
+      "shipping": {
+        "firstname": "John",
+        "lastname": "Doe",
+        "company": "Automattic",
+        "country": "United States (US)",
+        "addressfirstline": "addr 1",
+        "addresssecondline": "addr 2",
+        "city": "San Francisco",
+        "state": "CA",
+        "postcode": "94107"
+      }
+    }
```

Comment:
Yes, I think it is a good idea to include several profiles depending on where the store or customers are located but I would rather introduce them when we'd need them so it could be tested right away.

Reply:
Yup, was just pondering about how to add new test cases, seems like your solution supports it nicely.